airbyte-webapp/src/components/ColumnSortButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./ColumnSortButton.module.scss";
+import SortIcon from "./SortIcon";
+
+interface ColumnSortButtonProps {
+  onClick: () => void;
+  formattedMessageId: string;
+  wasActive: boolean;
+  lowToLarge: boolean;
+}
+
+export const ColumnSortButton: React.FC<ColumnSortButtonProps> = ({
```

Comment:
If we're using the children as suggested below, in order to make this compatible with React 18:

```suggestion
export const ColumnSortButton: React.FC<React.PropsWithChildren<ColumnSortButtonProps>> = ({
```